src/smtp.rs

```diff
@@ -753,6 +753,7 @@ async fn send_mdn_msg_id(
 }
 
 /// Tries to send a single MDN. Returns false if there are no MDNs to send.
```

Comment:
Maybe remove this "Returns..." sentence then?

Reply:
or be more clear "Returns false if there are no MDNs to send. Returns true if more MDNs should be sent." (if that is true, if course :)